performance-metrics/src/main.rs

```diff
@@ -146,6 +146,15 @@ impl PerformanceTest {
             metrics.push((self.func_ptr)(&self.control));
         }
 
+        /*
+         * As ethr is not stable on arm, there may get no output from
+         * the test, in this case, an invalid value of "-1.0" is
+         * injected, so we need eliminate these incorrect values here
+         * before use the vector.
+         */
+        #[cfg(target_cfg = "aarch64")]
+        metrics.retain(|&x| x >= 0.0);
```

Comment:
I am a bit worried that it will introduce instability (with no valid value to calculate mean/std_dev) or generate misleading result (with very few valid values) for the aarch64 metrics CI. Not sure whether it is safer/better to disable the network latency test on aarch64 for now before we find a stable alternative. I will let @MrXinWang and @michael2012z decide what to do.

Reply:
I agree with @likebreath's concern. Either we disable the latency test for `aarch64`, or we find a different tool (a stable one) to measure it.

Reply:
Ok, let's disable network latency test on arm currently.